MORKExample/README.md

```diff
@@ -0,0 +1,7 @@
+# MORKExample
+
+MORKExample is an example project that uses the MORK extensions to persist data to Rave.
+
+To run the example project, navigate to the MORKExample folder and run `pod install`. Open MORKExample.xcworkspace and run the project.
```

Comment:
Possibly - this may be due to my personal bias, but generally I read READMEs on Github as opposed to reading them in XCode / Sublime / etc.., in order to get the nice formatting. I want to differentiate clearly, since we also have a Podfile in the root folder.